next-sitemap.config.js

```diff
@@ -59,6 +59,7 @@ module.exports = {
     additionalSitemaps: [
       "https://www.ssw.com.au/people/sitemap-index.xml",
       "https://www.ssw.com.au/rules/sitemap-index.xml",
+      "https://www.ssw.com.au/archive/sitemap.xml",
```

Comment:
Can we follow the same format? (i.e */archive/sitemap-index.xml)

Reply:
Sure, just let me update my other PR

Reply:
Done - I'll need my PR approved first though
https://github.com/SSWConsulting/SSW.Website-v1-Progress/pull/135

Reply:
Approved that PR.

Reply:
Done - I moved https://www.ssw.com.au/archive/sitemap.xml to https://www.ssw.com.au/archive/sitemap-index.xml